src/services/billing/cost-management/cost-dashboard/widgets/CostByRegion.vue

```diff
@@ -157,8 +119,49 @@ export default {
             if (state.chart) state.chart.dispose();
         });
 
+        const setPieChartData = (chartData) => {
+            const _continentData = cloneDeep(continentData);
+            chartData.forEach((d) => {
+                const targetIdx = _continentData.findIndex(continent => continent.continent_code === (state.regions[`${d.provider}:${d.region_code}`]?.continent?.continent_code));
+                if (targetIdx !== -1) {
+                    _continentData[targetIdx].pieData.push({
+                        category: `${d.provider}:${d.region_code}`,
+                        value: d.usd_cost,
+                    });
+                }
+            });
+            return _continentData;
+        };
```

Comment:
```suggestion
        const setPieChartData = (chartData) => {
            const _continentData = cloneDeep(continentData);
            chartData.forEach((d) => {
                const target = _continentData.find(continent => continent.continent_code === (state.regions[`${d.provider}:${d.region_code}`]?.continent?.continent_code));
                if (target) {
                    target.pieData.push({
                        category: `${d.provider}:${d.region_code}`,
                        value: d.usd_cost,
                    });
                }
            });
            return _continentData;
        };
```

이렇게 하는건 어떻게 생각하시나요 ㅎㅎ findIndex 로 한번 더 타고 들어가는 뎁스를 줄이면 비교적 깔끔해보이지 않을까 싶어서요.